Description: Modify how fakenect is installed
 Install fakenect/README into share/doc as README.fakenect,
Forwarded: No
Author: arnebe <arne@alamut.de>, Nicolas Bourdaud <nicolas.bourdaud@gmail.com> 
Last-Update: 2012-01-128

--- a/fakenect/CMakeLists.txt
+++ b/fakenect/CMakeLists.txt
@@ -27,3 +27,6 @@ install (PROGRAMS ${CMAKE_CURRENT_BINARY
 
 install(FILES fakenect.1 fakenect-record.1
   DESTINATION ${PROJECT_MANPAGE_INSTALL_DIR}/man1)
+
+INSTALL(FILES "README" DESTINATION "share/doc/" RENAME "README.fakenect")
+
